conf.toml

```diff
@@ -43,3 +43,7 @@ HostOrchestratorPort = 1081
 
 [WebRTC]
 STUNServers = ["stun:stun.l.google.com:19302"]
+
+# Regex patterns of allowed origins
```

Comment:
Having regexes here is too permissive and could become a security issue. 

As we are only tackling CORS from **localhost** we could have an specific **`CORSAllowedLocalhost`** boolean config parameter that allows localhost only. On the other hand, if we need to allow external origins in the future we should list them fully rather than through regex matching.

Reply:
Changed to exact match.